staging/src/k8s.io/component-base/metrics/counter.go

```diff
@@ -107,26 +104,20 @@ func (c *Counter) initializeDeprecatedMetric() {
 
 // WithContext allows the normal Counter metric to pass in context.
 func (c *Counter) WithContext(ctx context.Context) CounterMetric {
-	c.ctx = ctx
-	return c.CounterMetric
-}
-
-// withExemplar initializes the exemplarMetric object and sets the exemplar value.
-func (c *Counter) withExemplar(v float64) {
-	(&exemplarCounterMetric{c}).withExemplar(v)
+	return &exemplarCounterMetric{ctx: ctx, delegate: c.CounterMetric}
 }
 
 func (c *Counter) Add(v float64) {
-	c.withExemplar(v)
+	c.CounterMetric.Add(v)
 }
 
 func (c *Counter) Inc() {
-	c.withExemplar(1)
+	c.CounterMetric.Add(1)
```

Comment:
CounterMetric has an `Inc()` that can be used here

Reply:
fixed